engine/swarm/configs.md

```diff
@@ -612,3 +612,53 @@ configuration file.
 
 You have now updated your `nginx` service's configuration without the need to
 rebuild its image.
+
+### Example: Use templated config in Nginx service
```

Comment:
Perhaps we should;

- keep it short, and remove "in Nginx service" from the title (there's already some Nginx examples on the page, which could make it harder to distinguish them)
- I think we should move this example before the advanced example ("Advanced example: Use configs with a Nginx service")

```suggestion
### Example: Use a templated config
```